Controller-dropdown should not be displayed in Fragment wizard #7543

ashklianko opened the pull request:
- Adding inspection panels and inseret panel depending on type of page item; text and part fragments for example only going to have one available inspection panel and no insert panel at all

alansemenov reviewed (changes requested):
We need to preselect layout/part/text of the fragment when nothing is selected in the Page Component view of a fragment. If it's selected and then unselected, then controller stays correctly selected in the Inspect panel.  

![image](https://github.com/enonic/app-contentstudio/assets/11612490/b53735c7-9c1d-4a7a-941c-6f477a27e2cf)

ashklianko committed:
Controller-dropdown should not be displayed in Fragment wizard #7543

- Adding inspection panels and inseret panel depending on type of page item; text and part fragments for example only going to have one available inspection panel and no insert panel at all

ashklianko commented:
Updated!

Commit:
ui-test issue #7553
